test/e2e/fixtures/DataGrid/PortaledEditCell.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { DataGrid } from '@material-ui/data-grid';
+
+const baselineProps = {
+  rows: [
+    {
+      id: 0,
+      brand: 'Nike',
+    },
+    {
+      id: 1,
+      brand: 'Adidas',
+    },
+    {
+      id: 2,
+      brand: 'Puma',
+    },
+  ],
+  columns: [
+    {
+      field: 'brand',
+      width: 100,
+      editable: true,
+      renderEditCell: ({ id, field, api }) =>
+        ReactDOM.createPortal(
```

Comment:
This test case looks strange. 

- Why not use the `<Portal>` component as in the other test cases?
- Why not use a regular test case, I mean why does it need a more costly e2e test? Is it about the click event sequence? (Sebastian added a helper in the core for it)
- Why not use the Select component directly?

Reply:
> Why not use the <Portal> component as in the other test cases?

Updated. I didn't remember of it.

> Why not use a regular test case, I mean why does it need a more costly e2e test? Is it about the click event sequence? (Sebastian added a helper in the core for it)

Yes, I chose to do an e2e test because of the click sequence. The helper is not merged yet: https://github.com/mui-org/material-ui/pull/25741/files#diff-0468d4d0b40c7b10080399740c336c59638e96c9e10196e9537ea4fda5a80646. 

> Why not use the Select component directly (for a second test case of integration)?

I updated the PR to do an e2e test with Select and a regular test case with Portal.